sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -166,6 +166,8 @@ async def read(
         :keyword bool populate_quota_info: Enable returning collection storage quota information in response headers.
         :keyword str session_token: Token for use with Session consistency.
         :keyword dict[str, str] initial_headers: Initial headers to be sent as part of the request.
+        :keyword list[str] excluded_locations: Excluded locations to be skipped from preferred locations. The locations
+            in this list are specified as the names of the azure Cosmos locations like, 'West US', 'East US' and so on.
```

Comment:
this is a metadata request (container.read()) - not sure if excluded regions would apply here